src/main/java/com/sublinks/sublinksapi/comment/repositories/LinkPersonCommentRepository.java

```diff
@@ -0,0 +1,36 @@
+package com.sublinks.sublinksapi.comment.repositories;
+
+import com.sublinks.sublinksapi.comment.entities.Comment;
+import com.sublinks.sublinksapi.comment.entities.LinkPersonComment;
+import com.sublinks.sublinksapi.comment.enums.LinkPersonCommentType;
+import com.sublinks.sublinksapi.person.entities.Person;
+import java.util.List;
+import java.util.Optional;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface LinkPersonCommentRepository extends JpaRepository<LinkPersonComment, Long> {
+
+
+  Optional<LinkPersonComment> getLinkPersonCommentByCommentAndPersonAndLinkType(Comment comment,
```

Comment:
I forget that Java supports method overloading. Do we really need to have all of these methods have different names? Why not just have `getLinkPersonComment` and multiple different signatures?

Reply:
Because this is jpa? Or i could of course make custom query with @Query but then we will have to do this on every other repository too.

Reply:
Something to discuss with @jgrim I think